pepper-apis/dss-core/src/main/java/org/broadinstitute/ddp/util/DataLoaderMain.java

```diff
@@ -249,11 +247,11 @@ public void processGoogleBucketParticipantFiles(Config cfg, String studyGuid, bo
         for (Blob file : bucket.list().iterateAll()) {
             data = new String(file.getContent());
             if (!file.getName().startsWith("Participant_") && !file.getName().startsWith("LovedOne_")) {
-                LOG.info("Skipping bucket file: {}", file.getName());
+                log.info("Skipping bucket file: {}", file.getName());
                 continue;
                 //not a participant data .. continue to next file.
             }
-            LOG.info("File Name: {} \n DATA:\n", file.getName(), data);
+            log.info("File Name: {} \n DATA:\n", file.getName(), data);
```

Comment:
The `data` object is unused, let's remove it.
```suggestion
            log.info("File Name: {}", file.getName());
```